### ISSUE TITLE
Gelatinous ascites

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10108]
**Preferred term label:**
Gelatinous ascites

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0033822

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pseudomyxoma peritonei

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000068
- label: Gelatinous ascites
- parents: [Ascites (HP:0001541)](https://hpo.jax.org/app/browse/term/HP:0001541)
- pmid: PMID:28868432
- def: A type of ascites characterized by copious amounts of mucinous ascites and mucinous peritoneal implants, leading to progressive obliteration of the peritoneal cavity and intestinal obstruction.
- comment: See figures 2 and 3 of PMID:28868432.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10108
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
correcting HPO id to HP:0040414
